catchpy/requirements/aws.txt

```diff
@@ -0,0 +1 @@
+-r base.txt
```

Comment:
what is your use case that needs this peculiar location of the requirements file?

Reply:
Good question @nmaekawa , this is a peculiarity of our deployment conventions. We usually set the settings module to **aws** (e.g. `{{ project_name }}.settings.aws`) and then load a requirements file named after it (e.g. `{{ project_name }}/requirements/aws.txt`). Since we have dev, qa, stage, and prod environments, we just point to that settings module and requirements file for all tiers (don't have to worry about getting the correct one for that env), and maintain a separate one for local/developer environments and maybe another for CI testing if needed. The other differences are separated out via the secure settings configuration (e.g. dotenv). 

We may be able to change it on our end so it looks for the root `requirements.txt` instead (or whatever) and then the `catchpy/requirements/aws.txt` could be eliminated from this PR. Let me know if you'd like me to explore that option. I did it this way since it's the path of least resistance.